index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>Easy Science</title>
+  <script src="script.js" defer></script>
+</head>
+<body>
+  <header>
+    <section class="social-network">
+      <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">Español</a></li>
+      </ul>
+    </section>
+    <nav>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <div class="hamburger-menu"><img id="menu-button" src="assets/hamburger-menu.png" alt="menu"></div>
+      <ul id="menu">
+        <li class="closeButtonContainer"><span id="closeButton">X</span></li>
+        <li><a class="Personal-information" href="courses.html">Courses</a></li>
+        <li><a class="Personal-information" href="#about">About</a></li>
+        <li><a class="Personal-information" href="#contact">Contact</a></li>
+        <li><a class="Personal-information button sign-up" href="#">Sign up</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="home-page">
+        <div class="home-text">
+          <h1>We teach science the right way</h1>
+          <p>We have the mission to provide a world-class education for anyone, anywhere.</p>
+          <a class="button" href="courses.html">Start here</a>
+        </div>
+    </section>
```

Comment:
- [ ] [REQUIRED] You did a very good job here, but this top section is meant to have a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. I also suggest that it exactly follows the design below 👇🏻 

![image](https://user-images.githubusercontent.com/4247740/147304458-01f6fba1-285e-4d40-80b6-d14ffb27f3e5.png)